ts/features/services/home/components/FeaturedServicesCarousel.tsx

```diff
@@ -0,0 +1,74 @@
+import {
+  HSpacer,
+  IOSpacingScale,
+  IOVisualCostants
+} from "@pagopa/io-app-design-system";
+import * as React from "react";
+import { FlatListProps, StyleSheet } from "react-native";
+import { FlatList } from "react-native-gesture-handler";
+import { WithTestID } from "../../../../types/WithTestID";
+import {
+  FeaturedServiceCard,
+  FeaturedServiceCardProps,
+  FeaturedServiceCardSkeleton
+} from "./FeaturedServiceCard";
+
+export type FeaturedServicesCarouselProps = WithTestID<{
+  services: Array<FeaturedServiceCardProps>;
+}>;
+
+const CARD_SPACING: IOSpacingScale = 8;
+const CARD_WIDTH: number = 100;
```

Comment:
Probably better to import this value from the `FeaturedServiceCard`?
```suggestion
```

Reply:
fe840b861